src/dodal/devices/i10/i10_apple2.py

```diff
@@ -270,22 +284,22 @@ async def set(self, value: float) -> None:
             )
         # Moving to real angle which is 210 to 30.
         alpha_real = value if value > self.angle_threshold_deg else value + 180.0
-        jawphase = self.jawphase_from_angle(alpha_real)
-        if abs(jawphase) > self.jawphase_limit:
+        jaw_phase = self.jaw_phase_from_angle(alpha_real)
+        if abs(jaw_phase) > self.jaw_phase_limit:
             raise RuntimeError(
-                f"jawphase position for angle ({value}) is outside permitted range"
-                f" [-{self.jawphase_limit}, {self.jawphase_limit}]"
+                f"jaw_phase position for angle ({value}) is outside permitted range"
+                f" [-{self.jaw_phase_limit}, {self.jaw_phase_limit}]"
             )
-        await self.id.id_jaw_phase.set(jawphase)
+        await self.id.id_jaw_phase.set(jaw_phase)
         self._angle_set(value)
 
 
 def convert_csv_to_lookup(
     file: str,
     source: tuple[str, str],
-    mode: str = "Mode",
-    min_energy: str = "MinEnergy",
-    max_energy: str = "MaxEnergy",
+    mode: str | None = "Mode",
+    min_energy: str | None = "MinEnergy",
+    max_energy: str | None = "MaxEnergy",
```

Comment:
should: consider if all lowercase would be more robust. if it's coming from the csv it might be more difficult